### PR TITLE
fixed #3 database setup

### LINKED ISSUE #3: Database Setup

### PR DESCRIPTION
**Pull request by Sondos-Alalami**
fixed  #3
-  Adding connection file to database folder to allow connection to the database.
-  Creating build.sql file to create tables of the database with their attributes.
-  Creating build.js file to build a database.
-  Adding Postgres URLs to .env files.
-  Adding travis.yml file.

### CODE REVIEW

**Review by Ahmed-aed — DISMISSED** [review]
good job :+1:

**Review by AlaaShurrab — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+before_script:
+ - psql -c "create database db_test;" -U postgres
+ - psql -c "create user db1_project with password 'test';" -U postgres
+env: 
+ global:
+ - PGPORT=5432
+test:
+ adapter: postgresql
+ database: db_test
```
Comment (mFlaifel) [inline .travis.yml]: just add an empty line

File: server/app.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-require('dotenv').config();
+
```
Comment (mFlaifel) [inline server/app.js]: remove this empty file

File: server/database/data/build.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+const {join} = require('path');
+const {readFileSync} = require('fs');
+
+const buildDb = () => {
+  const buildSql = readFileSync(join(__dirname,'build.sql')).toString();
+  const fakeData = readFileSync(join(__dirname,'fakeData.sql')).toString();
+  return connection.query(buildSql).then(()=>connection.query(fakeData));
+};
+
+buildDb();
```
Comment (mFlaifel) [inline server/database/data/build.js]: we invoke this command in the test file not here.

**Review by mFlaifel — CHANGES_REQUESTED** [review]

**Review by mFlaifel — APPROVED** [review]
good job

**Review by Ahmed-aed — APPROVED** [review]

**Pull request merged**